miri-script/src/util.rs

```diff
@@ -146,3 +146,43 @@ impl MiriEnv {
         Ok(())
     }
 }
+
+/// Receives an iterator of files.
+/// Will format each file with the miri rustfmt config.
+/// Does not follow module relationships.
+pub fn format_files(
+    shell: &Shell,
+    files: impl Iterator<Item = Result<PathBuf, walkdir::Error>>,
+    toolchain: &str,
+    config_path: &Path,
+    flags: &[OsString],
+) -> anyhow::Result<()> {
+    use itertools::Itertools;
+
+    let mut first = true;
+    for batch in &files.chunks(128) {
+        // Build base command.
+        let mut cmd = cmd!(
+            shell,
+            "rustfmt +{toolchain} --edition=2021 --config-path {config_path} --unstable-features --skip-children {flags...}"
+        );
+        if first {
+            eprintln!("$ {cmd} ...");
+            first = false;
+        }
+        // Add files.
+        for file in batch {
+            // Make it a relative path so that we become immune to someone cloning the repo
+            // 50 directories deep
+            let file = file?;
+            let file = file.strip_prefix(shell.current_dir())?;
```

Comment:
```suggestion
            let file = file?.strip_prefix(shell.current_dir())?;
```

Reply:
This doesn't exactly work because `strip_prefix()` returns a `&Path`, not a `PathBuf` so the PathBuf is dropped at the end of the statement.